src/AppInput.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+class AppInput extends Component {
+  constructor(props){
+    super(props);
+    this.handleLocationTextInputChange =
+    this.handleLocationTextInputChange.bind(this);
+  }
+
+
+  handleLocationTextInputChange(e) {
+  this.props.onLocationTextInput(e.target.value);
+}
+
+render(){
+  return (
+    <div className="inputWrapper">
+    <h3>Enter a location to search Air Quality</h3>
+    <div className="searchArea">
+    <center>
```

Comment:
There is some weird indentation going on here.... We still try and follow the html tag indentation guide here (child elements are indented)